### DISCUSSION TITLE
GitHub上aedt文件如何下载?

### DISCUSSION THREAD

**Discussion by jiangmy97** [POLYU-EMLAB/polyu-emhub#17] [Q&A] [answered]
如题, github上aedt变成了raw文件,请问如何下载原始文件?

**Comment by hh-wu** [comment, marked as answer]
1. clone repo using git
2. 点击需要下载的文件链接，显示如下图。

![image](https://user-images.githubusercontent.com/58615099/131316106-e10dc326-5985-4b31-9835-a25ff7d7cc37.png)

然后右键点击download按钮，保存为.aedt文件

![image](https://user-images.githubusercontent.com/58615099/131316528-87233b31-21d0-43e7-9aa4-ccc99d1a2304.png)
